{
    "CVE-ID": "CVE-2018-1000117",
    "NVD": {
        "Descriptions": "Python Software Foundation CPython version From 3.2 until 3.6.4 on Windows contains a Buffer Overflow vulnerability in os.symlink() function on Windows that can result in Arbitrary code execution, likely escalation of privilege. This attack appears to be exploitable via a python script that creates a symlink with an attacker controlled name or location. This vulnerability appears to have been fixed in 3.7.0 and 3.6.5.",
        "Cvss2_BaseScore": 7.2,
        "Cvss3_BaseScore": 6.7,
        "Cvss2_VectorString": "AV:L/AC:L/Au:N/C:C/I:C/A:C",
        "Cvss3_VectorString": "CVSS:3.1/AV:L/AC:L/PR:H/UI:N/S:U/C:H/I:H/A:H",
        "CPEs": [
            {
                "URI": "cpe:/a:python:python:3.7.0:beta5",
                "FormattedString": "cpe:2.3:a:python:python:3.7.0:beta5:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python:3.7.0:beta4",
                "FormattedString": "cpe:2.3:a:python:python:3.7.0:beta4:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python:3.7.0:beta3",
                "FormattedString": "cpe:2.3:a:python:python:3.7.0:beta3:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python:3.7.0:beta2",
                "FormattedString": "cpe:2.3:a:python:python:3.7.0:beta2:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python:3.7.0:beta1",
                "FormattedString": "cpe:2.3:a:python:python:3.7.0:beta1:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python",
                "FormattedString": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "3.2.0",
                "VersionEndExcluding": "3.4.9",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python",
                "FormattedString": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "3.5.0",
                "VersionEndExcluding": "3.5.6",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python",
                "FormattedString": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "3.6.0",
                "VersionEndExcluding": "3.6.5",
                "VersionEndIncluding": ""
            }
        ]
    },
    "Official": {
        "3.4.9rc1": {
            "source": [
                "https://github.com/python/cpython/blob/3.4/Misc/NEWS.d/3.4.9rc1.rst"
            ],
            "BTS": {
                "bpo33001": {
                    "ID": "bpo33001",
                    "url": "https://bugs.python.org/issue33001",
                    "date": "2018-03-05-10-09-51",
                    "nonce": "elj4Aa",
                    "section": "Security"
                }
            }
        },
        "3.5.6rc1": {
            "source": [
                "https://github.com/python/cpython/blob/3.5/Misc/NEWS.d/3.5.6rc1.rst"
            ],
            "BTS": {
                "bpo33001": {
                    "ID": "bpo33001",
                    "url": "https://bugs.python.org/issue33001",
                    "date": "2018-03-05-10-09-51",
                    "nonce": "elj4Aa",
                    "section": "Security"
                }
            }
        },
        "3.6.5rc1": {
            "source": [
                "https://github.com/python/cpython/blob/v3.13.0a3/Misc/NEWS.d/3.6.5rc1.rst"
            ],
            "BTS": {
                "bpo33001": {
                    "ID": "bpo33001",
                    "url": "https://bugs.python.org/issue33001",
                    "date": "2018-03-05-10-09-51",
                    "nonce": "elj4Aa",
                    "section": "Security"
                }
            }
        },
        "3.7.0b3": {
            "source": [
                "https://github.com/python/cpython/blob/v3.13.0a1/Misc/NEWS.d/3.7.0b3.rst"
            ],
            "BTS": {
                "bpo33001": {
                    "ID": "bpo33001",
                    "url": "https://bugs.python.org/issue33001",
                    "date": "2018-03-05-10-09-51",
                    "nonce": "elj4Aa",
                    "section": "Security"
                }
            }
        },
        "3.8.0a1": {
            "source": [
                "https://github.com/python/cpython/blob/3.11/Misc/NEWS.d/3.8.0a1.rst"
            ],
            "BTS": {
                "bpo33001": {
                    "ID": "bpo33001",
                    "url": "https://bugs.python.org/issue33001",
                    "date": "2018-03-05-10-09-51",
                    "nonce": "elj4Aa",
                    "section": "Security"
                }
            }
        }
    }
}